app/controllers/renewal/InvolvedInOtherController.scala

```diff
@@ -54,25 +54,19 @@ class InvolvedInOtherController @Inject()(
         }
   }
 
-  private def businessTypes(activities: BusinessMatching): Option[String] = {
-    val typesString = activities.activities map { a =>
-      a.businessActivities.map {
-        case AccountancyServices => Messages("businessmatching.registerservices.servicename.lbl.01")
-        case BillPaymentServices => Messages("businessmatching.registerservices.servicename.lbl.02")
-        case EstateAgentBusinessService => Messages("businessmatching.registerservices.servicename.lbl.03")
-        case HighValueDealing => Messages("businessmatching.registerservices.servicename.lbl.04")
-        case MoneyServiceBusiness => Messages("businessmatching.registerservices.servicename.lbl.05")
-        case TrustAndCompanyServices => Messages("businessmatching.registerservices.servicename.lbl.06")
-        case TelephonePaymentService => Messages("businessmatching.registerservices.servicename.lbl.07")
+  private def businessTypes(activities: BusinessMatching): Option[List[String]] = {
+    val typesString: Option[List[String]] = activities.createBusinessTypeOrderedList
+
+    typesString.map {
+      case t =>
+        t.map(item => {
+        if (item.toLowerCase.startsWith("a") || item.toLowerCase.startsWith("e")) {
```

Comment:
Indent please

Reply:
@matthyatt1 @drewmiley refactoring is done